src/ocellus.erl

```diff
@@ -1,14 +1,26 @@
 -module(ocellus).
 -export([start/0]).
+-export([get_env/1, get_env/2]).
 
+-define(APP, ?MODULE).
+
+-spec start() -> ok.
 start() ->
-  ok = application:start(syntax_tools),
-  ok = application:start(compiler),
-  ok = application:start(goldrush),
-  ok = application:start(lager),
-  ok = application:start(crypto),
-  ok = application:start(asn1),
-  ok = application:start(public_key),
-  ok = application:start(ssl),
-  ok = application:start(inets),
-  ok = application:start(ocellus).
+    ok = application:start(syntax_tools),
```

Comment:
FYI I've started using `application:ensure_all_started(APP)` in `start()` functions now. It's way more convenient.